app/inventory.js

```diff
@@ -142,12 +142,22 @@ exports.findByAssetid = function (assetid) {
  * Returns all assetids with a matching sku
  * @param {String} sku
  * @param {Boolean} [includeInTrade=true]
+ * @param {Object} [dict=undefined] Optional, use when searching different inventory
  * @return {Array<Object>}
  */
-exports.findBySKU = function (sku, includeInTrade = true) {
-    const assetids = (dictionary[sku] || []);
+exports.findBySKU = function (sku, includeInTrade = true, dict = undefined) {
```

Comment:
Don't modify the function. The code will be changed out for classes later, but till then just use `dict[sku] || []`